src/app/components/Hero/Hero.jsx

```diff
@@ -3,16 +3,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Hero = ({ heroData }) => {
+  // Check if this list is one of "Staff Picks"
+  const isStaffPicksList = heroData.header === 'Staff Picks';
```

Comment:
Perhaps it would be better to use picksData.type as the test for determining if staff picks is being rendered rather than a hardcoded string.

Reply:
Agreed.

Reply:
Just seems more consistent to me to pass type down to the Hero component and test type === 'staff-picks' instead of the header's title.